demonstrations/tutorial_classical_expval_estimation.py

```diff
@@ -0,0 +1,530 @@
+r"""
+Classically estimating expectation values from parametrized quantum circuits
+============================================================================
+
+In the race between quantum and classical computing, an important question
+is whether there exist efficient classical algorithms to simulate quantum
+circuits.
+Probably the most widely-known result of this type is the Gottesman-Knill
+theorem [#gottesman]_. It states that quantum circuits consisting of Clifford
+gates alone can be simulated classically, provided that the initial state
+of the circuit is "nice enough" (also see :doc:`our demo on Clifford simulation
+</demos/tutorial_clifford_circuit_simulations>`).
+
+In this short demo we will look at a new result of this type by Angrisani
+et al. [#angrisani]_.
+Roughly, it provides an efficient classical algorithm to estimate expectation
+values of local observables for a large class of parametrized quantum circuits.
+Again, some conditions apply to which we will get later.
+
+First, we will focus on the main technique of this algorithm, called Pauli propagation,
+which is not unique to this algorithm, but is also used by related simulation
+algorithms [#aharonov]_, [#lowesa]_, [#begusic]_.
+We will introduce Pauli propagation mathematically and implement a basic variant
+of the algorithm in code.
+Afterwards, we will outline some of the important details behind the
+overly simplified statement "Parametrized quantum circuits can be simulated
+classically", which are the important core of the new result [#angrisani]_.
+
+Which computation are we going to simulate?
+-------------------------------------------
+
+Let's start by looking at the type of quantum computations that we want to
+simulate classically.
+Given an initial state :math:`|\psi_0\rangle`, a parametrized quantum
+circuit :math:`U(\theta)`, and an observable :math:`H`, a common task is
```

Comment:
```suggestion
circuit :math:`U(\theta)`, and an observable :math:`H`, a common task for this approach is
```
Or field, or problem. A common task where?